src/index.html

```diff
@@ -6,12 +6,167 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="http://localhost:3000/api"
```

Comment:
Fix the action here according to the requirements 
<img width="789" alt="image" src="https://user-images.githubusercontent.com/99403198/233781163-5bd39e63-f608-412b-95d5-ee5eb104afb2.png">

Reply:
I can`t open this link. I must insert instead this link in line action ?